internal/cmd/fetcher/main.go

```diff
@@ -25,9 +25,11 @@ import (
 )
 
 var (
-	bazelDownloadRegexp = regexp.MustCompile(`bazelbuild/bazel/releases/download/[^/]+/bazel-[^-]+-linux`)
-	bazelImageName      = "gcr.io/bazel-public/bazel"
-	errNoVersions       = errors.New("no versions found")
+	bazelDownloadRegexp   = regexp.MustCompile(`bazelbuild/bazel/releases/download/[^/]+/bazel-[^-]+-linux`)
+	bazelImageName        = "gcr.io/bazel-public/bazel"
+	errNoVersions         = errors.New("no versions found")
+	dockerSyntaxPrefix    = "# syntax=docker/dockerfile:"
+	preferredDockerSyntax = "1.9"
```

Comment:
Not quite, we should always be able to get this value from this file, which itself is kept up-to-date via dependabot.

https://github.com/bufbuild/plugins/blob/main/.github/docker/Dockerfile.dockerfile